docs/source/conf.py

```diff
@@ -55,17 +57,17 @@
 
 # General information about the project.
 project = u'rfc3986'
-copyright = u'2017, Ian Stapleton Cordasco'
+copyright = u'{}, Ian Stapleton Cordasco'.format(datetime.now().year)
```

Comment:
The way copyright works, you only really need the start year

https://github.com/python-trio/trio/pull/510#issuecomment-384577383

Reply:
Right. I'm not a copyright expert obviously ;-)

I was just trying to make the documentation look more recent.
Maybe this is an option?

`copyright = u'2017-{}, Ian Stapleton Cordasco'.format(datetime.now().year)`

Or leave it as it is?

Reply:
Please just leave this as it was. The range isn't necessary as I understand it either.